etcdctl/ctlv3/command/migrate_command.go

```diff
@@ -220,7 +220,7 @@ func writeKeys(w io.Writer, n *store.NodeExtern) uint64 {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
-	fmt.Fprintf(w, string(b))
+	fmt.Fprintln(w, string(b))
```

Comment:
Also this should be https://golang.org/pkg/fmt/#Fprint not https://golang.org/pkg/fmt/#Fprintln